common/update_inventory.yml

```diff
@@ -38,9 +38,5 @@
         add_host_in_memory_inventory_pwd: "{{ vm_password }}"
         add_host_in_memory_inventory_sudo_pwd: "{{ vm_password }}"
```

Comment:
It must be a mistake before, I think we can also remove this one "add_host_in_memory_inventory_sudo_pwd".

Reply:
Thanks. It is removed now.